lib/config/presets/internal/group.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { presets } from './group';
+
+const exceptions = new Set(['monorepos', 'recommended']);
+
+describe('config/presets/internal/group', () => {
+  it('presets should have right name', () => {
```

Comment:
You could use `it.each` to create a test per preset, which would show us all failing presets rather the first.